### ISSUE TITLE
Demucs accuracy got failed

### ISSUE THREAD

**Issue by mengfei25** [intel/torch-xpu-ops#488]
### 🐛 Describe the bug

torchbench_amp_fp16_training
xpu  train demucs
E0626 17:48:23.409000 140063059666752 torch/_dynamo/utils.py:1478] RMSE (res-fp64): 0.03316, (ref-fp64): 0.00065 and shape=torch.Size([]). res.dtype: torch.float32, multiplier: 3.000000, tol: 0.001000
fail_accuracy

### Versions

torch-xpu-ops: https://github.com/intel/torch-xpu-ops/commit/31c400195d63064940242220dc9100322d36bac4
pytorch: 0f81473d7b4a1bf09246410712df22541be7caf3 + PRs: 127277,129120
device: PVC 1100, 803.61, 0.5.1

**Comment by retonym** [comment]
This model failure is not the random seed setting issue which we fixed before.
It should be a new issue caused by recent changes. Previously, the model could pass after fixing random seed issue.

**Comment by retonym** [comment]
Pass with minify enabled, still suspect related to random seed issue

**Comment by retonym** [comment]
This issue is related to sort operator, will investigate the issue further.

**Comment by retonym** [comment]
Model can pass with https://github.com/pytorch/pytorch/pull/134302 to disable reorder_for_locality.
The random kernels output different value if their orders are changed on xpu.
